pkg/kubelet/kubelet.go

```diff
@@ -1731,6 +1796,78 @@ func (kl *Kubelet) getPullSecretsForPod(pod *api.Pod) ([]api.Secret, error) {
 	return pullSecrets, nil
 }
 
+// listPodsFromCgroupMounts scans through all the subsytems of pod cgroups
+// Get list of pods whose cgroup still exist on the cgroup mounts
+func (kl *Kubelet) listPodsFromCgroupMounts() (map[types.UID]string, error) {
```

Comment:
Yes I planned on moving all the cgroup deletion specific logic into ContainerManager, but refrained from it as I wanted to get this working as soon as possible and refactor later on.
If you wd like I can refactor the code in this PR itslelf.